File: app/src/main/java/me/larikraun/tourreviews/dagger/RxCallAdapterFactoryModule.java
package me.larikraun.tourreviews.dagger;

import dagger.Module;
import dagger.Provides;
import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory;

@Module
public class RxCallAdapterFactoryModule {
	
	@Provides
	public RxJava2CallAdapterFactory provideRxJavaCallAdapterFactory () {
		return RxJava2CallAdapterFactory.create ();
	}
}

File: app/src/main/java/me/larikraun/tourreviews/dagger/RxCallAdapterFactoryModule.kt
package me.larikraun.tourreviews.dagger

import dagger.Module
import dagger.Provides
import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory

@Module
class RxCallAdapterFactoryModule {

    @Provides
    fun provideRxJavaCallAdapterFactory(): RxJava2CallAdapterFactory {
        return RxJava2CallAdapterFactory.create()
    }
}
